org.eclipse.dartboard/src/org/eclipse/dartboard/preference/DartSDKLocationFieldEditor.java

```diff
@@ -1,14 +1,24 @@
 package org.eclipse.dartboard.preference;
 
-import java.io.File;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
+import org.eclipse.core.runtime.Platform;
 import org.eclipse.dartboard.Messages;
 import org.eclipse.jface.preference.DirectoryFieldEditor;
 import org.eclipse.swt.events.ModifyListener;
 import org.eclipse.swt.widgets.Composite;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class DartSDKLocationFieldEditor extends DirectoryFieldEditor {
 
+	private static final Logger LOG = LoggerFactory.getLogger(DartSDKLocationFieldEditor.class);
```

Comment:
We need to remove once logger change merged.

Reply:
Of course, whatever is merged first.